pkg/envoy/boilerplate.go

```diff
@@ -56,9 +59,18 @@ func init() {
 		log.Fatal(err)
 	}
 	jsonFormat = b.GetStructValue()
+
+	allowedRetryOns := "5xx|gateway-error|reset|connect-failure|envoy-ratelimited|retriable-4xx|refused-stream|retriable-status-codes|retriable-headers|http3-post-connect-failure"
```

Comment:
I'd rather convert this to an set (map of bool of empty interfaces) of strings to make it simpler to add more policies and for quick lookup

Reply:
Thx for the suggestion! Fixed.